config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -129,14 +129,14 @@ periodics:
     description: "Run serial node e2e tests on ARM64 environment on Ubuntu"
   decorate: true
   extra_refs:
-  - org: kubernetes
-    repo: kubernetes
-    base_ref: master
-    path_alias: k8s.io/kubernetes
   - org: kubernetes
     repo: test-infra
     base_ref: master
     path_alias: k8s.io/test-infra
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
```

Comment:
just in case there is anything with the ordering, might not necessary.

Reply:
this indeed is not necessary, please revert this ordering